examples/svsm.rs

```diff
@@ -102,8 +100,8 @@ fn main() {
     attestation.measurement[0] = 42;
     attestation.measurement[47] = 24;
 
-    let cr = ClientRegistration::new(workload_id.clone());
```

Comment:
Should we put the ID in the `AttestationReport`?

Reply:
The host will already do that when launching the guest.

Reply:
`workload_id` is only used in reference-kbs. keybroker has no use for a `workload_id`.

Reply:
i.e. I'd put the `reference_id` returned by the registration (first step) in the attestation, just to test the protocol till the attestation.
Anyway, we can do also later ;-)